roles/velero/templates/registry_config_map.yml.j2

```diff
@@ -0,0 +1,18 @@
+---
+{% for bsl in backup_storage_locations %}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: registry-config
+  namespace: {{ velero_namespace }}
+data:
+{% if bsl.provider == 'aws' and velero_aws_bsl_configs | int != 0 %}
```

Comment:
I'm worried this won't be enough to support multiple BSLs that use AWS. We cannot use a hardcoded name for the route, will likely need to let Kubernetes generate the route name and register it to be stored in the configmap.

Reply:
Used the combination of `<bls.name>-<bsl.provider>`  for creation of registry, svc, routes as well as the component label so that the naming remains unique.